src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -612,16 +612,24 @@ static int server_send_offer_or_ack(
         return dhcp_server_send_packet(server, req, packet, type, offset);
 }
 
-static int server_send_nak(sd_dhcp_server *server, DHCPRequest *req) {
+static int server_send_nak_or_ignore(sd_dhcp_server *server, bool init_reboot, DHCPRequest *req) {
         _cleanup_free_ DHCPPacket *packet = NULL;
         size_t offset;
         int r;
 
+        if (!init_reboot)
```

Comment:
Feel free to ignore, but to me it seemed better with this condition checked outside of the function - it doesn't seem very related. But not too important.

Reply:
I'd like to keep this as is. This makes `dhcp_server_handle_message()` simpler.

Instead, I added a brief comment why we silently ignore the request in that case.